Software/src/main/java/org/foi/emp/carmanagement/services/EmailSenderService.java

```diff
@@ -0,0 +1,24 @@
+package org.foi.emp.carmanagement.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EmailSenderService {
+
+    @Autowired
+    private JavaMailSender javaMailSender;
+
+    public void sendSimpleEmail(String toEmail, String body, String subject){
```

Comment:
Change this 
`public void sendSimpleEmail(String toEmail, String body, String subject)` to
`public void sendSimpleEmail(final String toEmail, final String body, final String subject)`

final operator is used at variable which is immutable, once it is assigned, his value cannot be changed.